sdk/python/kfp/client/client.py

```diff
@@ -689,9 +689,22 @@ def _choose_pipeline_file(file_list: List[str]) -> str:
                 f'The package_file {package_file} should end with one of the '
                 'following formats: [.tar.gz, .tgz, .zip, .yaml, .yml].')
 
-    def _override_caching_options(self, workflow: str,
+    def _override_caching_options(self, pipeline_obj: dict,
                                   enable_caching: bool) -> None:
-        raise NotImplementedError('enable_caching is not supported yet.')
+        """Overrides caching options.
+
+        Args:
+            pipeline_obj (dict): Dict object parsed from the yaml file.
+            enable_caching (bool): Overrides options, one of 'True', 'False'.
+        
+        Returns:
```

Comment:
nit: you can skip this when return is None.

https://google.github.io/styleguide/pyguide.html#doc-function-returns